.babelrc

```diff
@@ -4,7 +4,8 @@
   ],
   "plugins": [
     "macros",
-    "add-react-displayname"
+    "add-react-displayname",
+    "babel-plugin-styled-components"
```

Comment:
went ahead and added the babel-plugin thinking it might help with the props being passed down as DOM attributes issue, but it didn't 🤷‍♀️ There are [lots of other perks](https://www.styled-components.com/docs/tooling#babel-plugin) from adding the plugin though, so keeping it in :)

Reply:
I'm all for adding the babel plugin 👍 This would allow us to use the `css` prop if we want 😍